articles/ctod.dd

```diff
@@ -859,8 +859,8 @@ $(H4 The C Way)
        C initializes arrays by positional dependence. C99 fixes the issue:
 $(CCODE
```

Comment:
Can we used a fenced C code block here?

    ```C
    int a[3] = { 3,2,1 };
    /* ... */
    ```

Reply:
Ditto for the rest of the `$(CCODE ..)` changes.

Reply:
We certainly could, although we'd use a lowercase `c` to work with the existing CSS. The reason I didn't switch them  is because I wanted to minimize the amount of changes, and if we change one we'd probably want to change them all.